tests/cypress/support/commands.js

```diff
@@ -351,3 +351,20 @@ Cypress.Commands.add('waitForDocumentUpdate', (timeout=5000) => {
     cy.waitUntil(() => { return cy.document().its('lastModified').then($newUpdate => $newUpdate != lastUpdate) }, {'interval': 200, 'timeout':timeout})
   })
 })
+
+// To check the no resoures icon and message
+Cypress.Commands.add('checkPolicyNoResourcesIconMessage', (present=true, message='Resource not found') => {
+  cy.get('#page')
```

Comment:
Is this needed? If you want to wait for the page content to load, maybe you could use different approach, like
`cy.waitForPageContentLoad()`
https://github.com/open-cluster-management/grc-ui/blob/main/tests/cypress/support/commands.js#L190